### ISSUE TITLE
Node version

### ISSUE THREAD

**Issue by bgadrian** [jshimko/meteor-launchpad#86]
Hello,

I see that you have to specify the node version, othewise it will use the hard-coded one on the Dockerfile (now 8.9.0)
But  the meteor deploy docs state 

> Depending on the version of Meteor you are using, you should install the proper version of node using the appropriate installation process for your platform. To find out which version of node you should use, run meteor node -v in the development environment, or check the .node_version.txt file within the bundle generated by meteor build. For example, if you are using Meteor 1.6, the version of node you should use is 8.8.1.
> If you use a mis-matched version of Node when deploying your application, you will encounter errors!

The scripts does this and I missed it? Or it can be an improvement to use the .nodeversion one as a default?

**Comment by jshimko** [comment]
@bgadrian Yeah, I suppose this could be updated to use the `node_version.txt` instead of the latest version of Node supported by Meteor.  The main hassle there though is that `node_version.txt` doesn't exist until _after_ the `meteor build` command has created a bundle.  I suppose that's not really a problem though.  I could just move the Node install script to later in the build process.  That would be nicer than having to manually specify custom versions with a build arg.

Thanks for bringing this up.  I'll take a look at adding it (unless you're feeling motivated to submit a PR!).

**Comment by bgadrian** [comment]
I think it can be done before, on localhost we can do `meteor node --version`, but I'm not sure if he knows to get the project version (based on the meteor package version from my project) and not the global meteor version. Altrough the safest solution I think it would be to check the build.
